assets/src/design-system/components/snackbar/snackbarMessage.js

```diff
@@ -162,9 +225,29 @@ const SnackbarMessage = ({
         aria-label={ariaLabel}
         size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
         hasAction={hasAction}
+        hasThumbnail={hasThumbnail}
       >
         {message}
       </Message>
+      {thumbnail && (
+        <ThumbnailWrapper hasAction={hasAction}>
+          <Thumbnail
+            src={thumbnail.src}
+            alt={thumbnail.alt}
+            status={thumbnail.status}
+          />
+          {thumbnail.status === THUMBNAIL_STATUS.SUCCESS && (
+            <ThumbnailIcon status={thumbnail.status}>
+              <CheckmarkSmall />
+            </ThumbnailIcon>
+          )}
+          {thumbnail.status === THUMBNAIL_STATUS.ERROR && (
+            <ThumbnailIcon status={thumbnail.status}>
+              <ExclamationOutline />
+            </ThumbnailIcon>
+          )}
```

Comment:
nit: since the thumbnail status has to be one of these, you could do the check in the `ThumbnailIcon` 
```
<ThumbnailIcon status={thumbnail.status}>
{thumbnail.status === THUMBNAIL_STATUS.SUCCESS ? <CheckmarkSmall /> : <ExclaimationOutline />}
</ThumbnailIcon>
```

Reply:
gonna do this, but leave the option to not pass a status and have no icon render and just have the thumbnail. It's not used in the designs rn, but keeps the component a lil more open to future use imo 🤷